app/controllers/posts_controller.rb

```diff
@@ -5,7 +5,28 @@ def index
   end
 
   def show
+    # user = User.find(params[:user_id])
+    # @post = user.posts.find(params[:id])
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes. Kindly remove that before you merge the PR.